.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+version: 2
+jobs:
+  build:
+    working_directory: /go/src/github.com/Clever/kinesis-to-firehose
+    docker:
+      - image: circleci/golang:1.8
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: cd /tmp/ && wget https://bootstrap.pypa.io/get-pip.py && sudo python get-pip.py
+      - run: sudo pip install --upgrade awscli && aws --version
+      - run: pip install --upgrade --user awscli
+      - run: echo 'github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==' >> ~/.ssh/known_hosts
+      - run: cd $HOME && git clone --depth 1 -v git@github.com:clever/ci-scripts.git && cd ci-scripts && git show --oneline -s
+      - run: make install_deps
+      - run: make build
+      - run: make test
+      - run: mkdir -p /tmp/test-results
+      - run: "make bench | tee /tmp/test-results/benchmarks.txt"
+      - store_artifacts:
+          path: /tmp/test-results
+          destination: benchmarks
+      - store_test_results:
```

Comment:
we should this remove this block. We don't have the right format, and also this will not work with CircleWorkflows.